src/components/PageElements/Elements/ReviewInlineInput.tsx

```diff
@@ -49,7 +49,8 @@ const ReviewInlineInput: React.FC<ReviewInlineInputProps> = ({
   const optionsMap = useOptionsMap()
   const options = isConsolidation ? optionsMap.consolidation : optionsMap.review
 
-  const submit = async () => {
+  const submit = async (e: any) => {
+    e.preventDefault()
```

Comment:
Required to fix issue caused by React upgrade.

Reply:
Would there possibly be an issue in reviewInlineNewResponse.tsx which uses the same code?

Reply:
That's a very good point, thanks for spotting. I'm not totally sure, I'll need to go check this.

Reply:
I've actually just found another problem when I tried to test this -- I can't actually review any questions that the applicant didn't answer, so can't make this method run.

Have made an issue: https://github.com/openmsupply/conforma-web-app/issues/1537

So I'll leave this as-is for now, but I'll make a note to double-check this when doing that one.